server/models/TreeV2.ts

```diff
@@ -0,0 +1,121 @@
+import log from 'loglevel';
```

Comment:
@aayushgauba why we need to add this TreeV2.ts?

Reply:
@dadiorchen Because we dont have the same functionality in V2 to filter by UUID